docs/user_guide/queue_config.md

```diff
@@ -262,9 +272,10 @@ The star "*" is the wildcard character and matches all users or groups.
 Duplicate entries in the lists are ignored and do not cause a parsing error.
 Specifying a star beside other list elements is not allowed.
 
-_maxapplications_ is an unsigned integer value, larger than 1, which allows you to limit the number of running applications for the configured user or group.
+_maxapplications_ is an integer value, larger than 1, which allows you to limit the number of running applications for the configured user or group.
 Specifying a zero maximum applications limit is not allowed as it would implicitly deny access.
-Denying access must be handled via the ACL entries.  
+Denying access must be handled via the ACL entries.
+_parent_ queue _maxapplications_ value must be larger than _parent_ queue _maxapplications_ value.
```

Comment:
This is not correct: the maxapplications for the user and or group must be smaller than the value set for the queue. However we do not enforce that at the moment. If we add a validation it will not be against the parent queue.

Reply:
Good to know. If we don't enforce, I will remove this line.

Reply:
Removed.